tests/test_load_current_adrs.py

```diff
@@ -0,0 +1,370 @@
+import re
+import subprocess
+from mock import patch
+from datetime import datetime
+from decimal import Decimal
+
+import pytest
+
+import manage
+from webservices import rest
+from webservices.legal_docs import DOCS_INDEX
+from webservices.legal_docs.current_adrs import parse_regulatory_citations, parse_statutory_citations
+from tests.common import TEST_CONN, BaseTestCase
+
+@pytest.mark.parametrize("test_input,case_id,entity_id,expected", [
```

Comment:
`parse_regulatory_citations` and `parse_statutory_citations` need not be tested again - they should not be `copied`, they should be refactored into a `utility` module and _reused_.